contracts/libraries/MultiSend.sol

```diff
@@ -25,8 +25,11 @@ contract MultiSend {
     ///                     data length as a uint256 (=> 32 bytes),
     ///                     data as bytes.
     ///                     see abi.encodePacked for more information on packed encoding
+    /// @notice This method is payable as delegatecalls keep the msg.value from the previous call
```

Comment:
Can this `value` be split? I mean, `multiSend` received `3Wei` and does three inner txs of `1Wei` each? If that's the case, what happens if there's remaining `eth`? Shouldn't we check that total `value` sent should be equal to `msg.value`?

Reply:
the remaining eth stays in the Safe . I would not check if all ETH is used, as this adds complexity. 

The primary purpose of adding payable is to deactivate the generation of the `msg.value > 0` check by solidity. As this is not something that this contract should have to worry about.

Reply:
Good to know, thanks